scripts/monorepo/runPublished.js

```diff
@@ -28,7 +28,7 @@ const lageArgs = [
   'run',
   ...argv,
   // default to verbose mode unless already/otherwise specified
-  ...(argv.some(arg => /^--(no-)?verbose/.test(arg)) ? [] : ['--verbose']),
+  ...(argv.some((arg) => /^--(no-)?verbose/.test(arg)) ? [] : ['--verbose']),
```

Comment:
**q:** how did this (format) change happen ? your manual action in editor?

Reply:
It happened with me also, @ling1726 fixed it in the #17250  so just rebasing master should fix.

Reply:
After the upgrade to storybook 6, prettier v2 was introduced. As we saw the `lint-staged` precommit hooks were using a different prettier version than `yarn format`.

@Hotell fixed this when he introduced single version policy for prettier